### PR TITLE
feat: add forkmon

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

**Comment by barnabasbusa** [comment]
CI fails as `service_name` is not yet available. Once https://github.com/kurtosis-tech/eth-network-package/pull/53 is merged, it should pass.

**Comment by h4ck3rk3y** [comment]
@barnabasbusa I have added service name, can you run this again?

**Comment by barnabasbusa** [comment]
@h4ck3rk3y its now passing.

File: src/package_io/parse_input.star [hunk 1]
```diff
@@ -172,7 +168,6 @@ def default_network_params():
 		"seconds_per_slot":				12,
 		"slots_per_epoch":				32,
 		"genesis_delay":				120,
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:170]: Can you set this to 10 as well?

**Review by h4ck3rk3y — APPROVED** [review]

File: src/package_io/parse_input.star (hunk 1 above)
Comment (h4ck3rk3y) [reply src/package_io/parse_input.star:170]: @barnabasbusa 120 is okay; you know best

File: src/package_io/parse_input.star [hunk 2]
```diff
@@ -90,9 +90,6 @@ def parse_input(input_args):
 	if result["network_params"]["genesis_delay"] == 0:
 		fail("genesis_delay is 0 needs to be > 0 ")
 
-	if result["network_params"]["capella_fork_epoch"] == 0:
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:93]: blocking this for a bit till we resolve this

**Pull request merged**